### PR TITLE
Add .travis.yml with building in Docker image

### PR DESCRIPTION
**Pull request by hryniuk**

### CODE REVIEW

**Comment by hryniuk** [comment]
Hey, @serin-delaunay, I think we can merge it :smiley:. It's complete IMO and freshly rebased on the master branch. It builds successfully and works on Arch Linux and Ubuntu now. It doesn't work in Docker container, because of #95, but it's out of scope of this PR.

It builds on Travis, only one assert is failing. It will be great to rebase other pull requests on it, fixing this assert meanwhile.

**Comment by hryniuk** [comment]
Check if [this change](https://github.com/Wangscape/Wangscape/pull/71/commits/b5e55b799e48c7685bd7afd813c02f0ac405651a) doesn't break build on Windows, please.

File: README.md [hunk 1]
```diff
@@ -45,7 +44,45 @@ Wangscape depends on these libraries:
 After installing them, you can compile Wangscape using either of the methods
 below.
 
-## CMake (Linux)
+## Docker
+
+There's a Docker image that can be used for building, based on Ubuntu, with all
+dependencies installed, so the only thing to do is executing in the Wangscape's
```
Comment (serin-delaunay) [inline README.md]: "executing in Wangscape's root directory"

Comment (serin-delaunay) [inline README.md]: Sorry, I meant "the only thing to do is execute in Wangscape's root directory".

**Comment by serin-delaunay** [comment]
I was getting merge conflicts because the `master` branch in my local repo isn't synced with `Wangscape/master`, and I forgot to change Github's suggested checkout command. Whoops! :D

This branch compiles, runs, and passes all tests in VS on Windows. Merged.

**Pull request merged**